carta/session.py

```diff
@@ -417,6 +417,43 @@ def clear_raster_scaling_reference(self):
         """Clear the raster scaling reference."""
         self.call_action("clearRasterScalingReference")
 
+    # VIEWER MODES
+    @validate(Boolean())
+    def set_viewer_mode(self, multiple=True):
+        """
+        Switch between single-panel mode and multiple-panel mode.
+
+        Parameters
+        ----------
+        multiple : {0}
+            Whether to use multiple-panel mode. Default is ``True``.
+        """
+        self.call_action("widgetsStore.setImageMultiPanelEnabled", multiple)
+
+    def previous_page(self):
+        """Go to previous page in viewer"""
+        self.call_action("widgetsStore.onPreviousPageClick")
+
+    def next_page(self):
+        """Go to next page in viewer"""
```

Comment:
One last thing: each of these docstrings should have a full stop at the end.

Reply:
Done in 85cccaf.